Wiki Rewrite: "How To Add Mods" (#126)
Fabulously-Optimized/wiki: how-to/add-mods/curseforge-app.md

---
hidden: true
---

# CurseForge App

{% tabs %}
{% tab title="Through the CurseForge App" %}
1. Open the **CurseForge App**
2. Select **Minecraft** from the grid or from the sidebar
3. In **My Modpacks**, click on **Fabulously Optimized**

> 4. Click on the **three dots** button

Reviewer: ```suggestion
4. Click on the **ⵗ** (three dots) button
```
May need a bigger font to be more visible, not sure
Can also try `⋮` or `⫶` (yes, they are different haha)

> 5. Select **Profile Options**

Reviewer: ```suggestion
5. Select **⋮☰ Profile Options**
```
I'd like to keep (or even add) the symbols/emoji where possible, because that helps people find the button.

6. Check the **Allow content management for this profile** checkbox
7. Click on **Done**

> 8. Click on the **Puzzle** button

Reviewer: ```suggestion
8. Click on **+ Add More Content**
```
(the UI changed)

> 9. Find the mod or mods you want to add and select them

Reviewer: ```suggestion
9. Find the mod or mods you want to add and click **Install**
```

> 10. Once installed, launch the instance. Minecraft should open up

Reviewer: ```suggestion
10. Once installed, launch the instance by clicking **Play**. Minecraft should open up
```

11. If you can see the newly installed content in the Mod Menu, you're done!
{% endtab %}

{% tab title="Through the CurseForge website" %}
1. Find the mod or mods you want to add from the [list of Fabric mods on CurseForge](https://www.curseforge.com/minecraft/search?gameVersionTypeId=4)

> 2. Click on **Install** next to the latest version compatible with Fabric and Minecraft. The CurseForge App should open

Reviewer: The website's global install button is actually pretty stupid so we should instruct users to use the Files tab:

```suggestion
2. Click on the mod details page, then **Files**
3. Find a file that is compatible with Fabric and the Minecraft version you're using, click on it
4. Click on **T̲ Install**. The CurseForge App should open
```
(of course fix the proceeding numbering too)

3. Click on **Select a profile**

> 4. Select **Fabulously Optimized**

Reviewer: ```suggestion
6. Select **Fabulously Optimized**
     * If you are not able to select it, you picked the wrong mod loader or Minecraft version. Try again on the website
```

5. Click on **Install**

> 6. Once installed, launch the instance. Minecraft should open up

Reviewer: ```suggestion
6. Once installed, launch the instance by clicking **Play**. Minecraft should open up
```
